controllers/controller_utils_test.go

```diff
@@ -184,46 +182,6 @@ func expectDeployment(t *testing.T, g *gomega.GomegaWithT, requests chan reconci
 	return deploy
 }
 
-func verifyUserSuppliedTLSConfig(t *testing.T, tls *solr.SolrTLSOptions, expectedKeystorePasswordSecretName string, expectedKeystorePasswordSecretKey string, expectedTlsSecretName string, needsPkcs12InitContainer bool) {
```

Comment:
Moved these TLS specific test helper functions to `solrcloud_controller_tls_test` as they are only used by TLS tests and now seem out of place in this file